### PR TITLE
[autohint] Enclose the AUTOHINTEXE path in quotes

### PR DESCRIPTION
**Pull request by Arnie97**

### CODE REVIEW

**Comment by miguelsousa** [comment]
@Arnie97 thanks, the patch looks good. There has been a [GitHub incident](https://blog.github.com/2018-10-21-october21-incident-report/) that impacts this PR. We're waiting for the status to go green before merging.

File: python/afdko/autohint.py [hunk 1]
```diff
@@ -1237,7 +1237,7 @@ def hintFile(options):
 		else:
 			if os.path.exists(tempBezNew):
 				os.remove(tempBezNew)
-			command = AUTOHINTEXE + " %s%s%s%s -s %s -f \"%s\" \"%s\"" % (verboseArg, suppressEditArg, supressHintSubArg, decimalArg, NEWBEZ_SUFFIX, tempFI, tempBez)
+			command = '"%s" %s%s%s%s -s %s -f "%s" "%s"' % (AUTOHINTEXE, verboseArg, suppressEditArg, supressHintSubArg, decimalArg, NEWBEZ_SUFFIX, tempFI, tempBez)
```
Comment (anthrotype) [inline python/afdko/autohint.py]: if you called the subprocess with a list of strings instead a single string, then subprocess would take care of the quoting for you

Comment (miguelsousa) [reply python/afdko/autohint.py]: `autohint.py` is on the chopping block

Comment (miguelsousa) [reply python/afdko/autohint.py]: and `fdkutils.runShellCmd` is used in many places; any changes to it need to be done carefully.

**Pull request merged**